### PR TITLE
adp vignette/docs; ENS file/ignore

### PR DESCRIPTION
**Pull request by j-harbin**
This pull request updates the adp vignette to include information + an example about ship-based adp data. It also includes a .ENS file in the top directory (the .Rbuildignore has been updated to avoid any size problems). Lastly, it updates the adp docs to state that the adp dataset is a moored dataset.

### CODE REVIEW

**Comment by dankelley** [comment]
1. I may have already submitted comments ... I got turned around in the GH interface (as usual!).
2. Any chance you could spell-check your text, @j-harbin?  ... "recieved" doesn't quite look right :-).
3. I think your solution should have a `download.file()`, so that readers (like me) can try this out.  Also, you don't want to be stating your username, I think, so I'd avoid doing that.  Note that you can always download a file to a temporary directory, but I would instead suggest telling users to download to (some file) and then read that file.
4. I forgot that we had a function to remove the ship velocity.  That's sweet!  Oce is cool :-)

File: vignettes/adp.Rmd [hunk 1]
```diff
@@ -87,6 +87,8 @@ with `f` as defined above. This also illustrates that it is possibly to specify
 the location of an instrument; the further processing that was done to create
 `data(adp)` will be explained in due course.
 
++**Note**: User's who are not familiar with ADP data may noticed that there is a combination of many file types when dealing with ADP data. Some of these include: .ENR, which are raw data only recieved from the ADP, .ENS, similar to .ENR but also contain data merged from other file types without processing, .ENX, merged data that are processed including calculations to transformed data including east, north, and vertical velocities, .STA, which are a short term average of measurements, and .LTA, which are a long term average of measurements.
```
Comment (dankelley) [inline vignettes/adp.Rmd:90]: `User's` should be `Users`

**Comment by j-harbin** [comment]
I have spell checked, and created a folder called extras in the top directory that stores the .ENS (while updating the .Rbuildignore accordingly). Please ignore the following commit. I still need to fix the code for `download.files`.

File: vignettes/adp.Rmd [hunk 2]
```diff
@@ -114,6 +116,8 @@ Note that the summary reveals that instrument had been set up to record in
 **Exercise 1.** Determine the name of the `metadata` item that holds the
 coordinate system of an adp object.
 
+After completing exercise 1, the user will notice that there is both `latitude` and `longitude` available in the metadata for this built in, moored, adp dataset. It should be noted that if this dataset was a ship-based adp, `latitude` and `longitude` would not be included. Instead, the data would include, `firstLatitude`, `firstLongitude`, `lastLatitude`, and `lastLongitude` to account for the drift of the ship during sampling.
```
Comment (dankelley) [inline vignettes/adp.Rmd:119]: s/is both/both/

**Comment by j-harbin** [comment]
The most recent commit (commit b749bde420b06349f40c38232bb6d0594a4500ee) for the PR is the final version.

**Comment by dankelley** [comment]
Thanks, @j-harbin!  I've merged this and will take a look in a moment.  It fails one of the automated tests, but that failure seems to be unrelated to oce.  (These automated tests fail off an on, for reasons of server setup, I think.  The error message is saying it could not install R.  Well, that's not our fault!)

**Pull request merged**